src/main/java/com/dao/UserDaoImpl.java

```diff
@@ -40,11 +39,20 @@ public void add(User user) {
         }
     }
 
+    @Override
+    public User get(Long id) {
```

Comment:
let's use session get here, that will be shorter

Reply:
done